tests/GeoNames/ClientTest.php

```diff
@@ -46,6 +46,66 @@ public function testGetSupportedEndpoints()
         $this->assertContains('wikipediaSearch', $endpoints);
     }
 
+    public function testGetLastTotalResultsCountAssertNull()
+    {
+
+        // check for current value
+        $total = $this->client->getLastTotalResultsCount();
+
+        // make sure it's currently null
+        $this->assertNull($total);
+    }
+
+    public function testGetLastTotalResultsCountOnLargeResult()
```

Comment:
```suggestion
    public function testGetLastTotalResultsCountWhenResultIsLarge()
```

Reply:
I don't really care about this, but to me "when" is time-based. Otherwise it should rather be "If" or "InCaseOf" but actually, I find "On" much smoother ...